api/v3/clustermaster_types.go

```diff
@@ -64,6 +64,9 @@ type ClusterMasterStatus struct {
 
 	// App Framework status
 	AppContext AppDeploymentContext `json:"appContext"`
+
```

Comment:
Should we use Annotations for Apps, Apps are not standard configurations that remain constant , they may be optional, or they differ based on customer setup

Reply:
Hi @vivekr-splunk , in this case the Splunk Operator telemetry app is a static standard configuration. Could you provide an example of how annotations can be added to apps?